packages/components/src/tag/src/Tag.tsx

```diff
@@ -58,9 +70,10 @@ function Tag(props: TagProps, ref: ForwardedRef<HTMLDivElement>) {
         children: childrenProp,
         isInvalid,
         isLoading,
-        size: sizeProp = "md",
+        size: sizeProp,
```

Comment:
is it normal to have removed the default value?

Reply:
yes, if you look down, it's being set already. Other components only have it set down here like this:
`const size = useResponsiveValue(sizeProp) ?? "md";`

Reply:
I hadn't seen 👍